cppcache/src/statistics/GeodeStatisticsFactory.cpp

```diff
@@ -61,8 +61,9 @@ GeodeStatisticsFactory::~GeodeStatisticsFactory() {
     statsTypeMap.clear();
 
   } catch (const Exception& ex) {
-    Log::warningCatch("~GeodeStatisticsFactory swallowing Geode exception", ex);
-
+    LOGWARN(std::string(
```

Comment:
What is the rational for removing the a method that encapsulated the deconstruction of an exception being logged?